model/vowpalWabbit/dffml_model_vowpalWabbit/vw_base.py

```diff
@@ -373,6 +394,57 @@ async def predict(
 
 @entrypoint("vwmodel")
 class VWModel(Model):
+    """
+    Implemented using Vowpal Wabbit.
+
+    First we create the training and testing datasets
+
+    .. literalinclude:: /../model/vowpalWabbit/examples/train_data.sh
+
+    .. literalinclude:: /../model/vowpalWabbit/examples/test_data.sh
+
+    Train the model
+
+    .. literalinclude:: /../model/vowpalWabbit/examples/train.sh
+
+    Assess the accuracy
+
+    .. literalinclude:: /../model/vowpalWabbit/examples/accuracy.sh
+
+    Output
+
+    .. code-block::
+
+        0.38683876649129145
+
+
+    Make a prediction
+
+    .. literalinclude:: /../model/vowpalWabbit/examples/predict.sh
+
+    Output
+
+    .. code-block:: json
+
+        [
+            {
+                "extra": {},
+                "features": {
+                    "A": "| price:.46 sqft:.4 age:.10 1924"
```

Comment:
What's up with this?

Reply:
This is the standard input format taken by vowpal wabbit. So a feature can be a string in the specific format or a bunch of columns that get converted to this format if `convert_to_vw` flag is true.

Reply:
This shouldn't be the way we store that within a record though. It looks to me like this should be

```python
                "features": {
                    "price": 46,
                    "sqft": 4,
                    "age":10,
                }
```

Then it's unclear to me what `1924` is all about. The point here is that the features should be stored in the standard format when they're in the `Record` and if `convert_to_vw` flag is needed to change them into the vw format, then that should probably default to True.

I feel like I may be missing something here though?